etc/ansible/roles/network-runner/providers/dellemc.os9.os9/conf_access_port.yaml

```diff
@@ -0,0 +1,23 @@
+---
+- name: "dellemc.os9: reset interface to default"
+  dellemc.os9.os9_config:
+    lines:
+      - "default interface {{ port_name }}"
+  connection: network_cli
+
+- name: "dellemc.os9: enable switchport, rstp, and jumbo frames"
+  dellemc.os9.os9_config:
+    lines:
+      - "interface {{ port_name }}"
+      - "switchport"
+      - "no shutdown"
+      - "spanning-tree rstp edge-port"
+      - "mtu 9216"
```

Comment:
It defaults to the standard MTU which is 1500. The only reason I noticed this is because on all switchports in the OCT racks they had the MTU set to 9216.

Reply:
maybe we can make that conditional on some variable, just like how you suggested we do stp.